tests/matrix_free/laplace_operator_01.cc

```diff
@@ -108,7 +108,7 @@ void test ()
   }
 
   MatrixFreeOperators::LaplaceOperator<dim,fe_degree,fe_degree+1, 1, number> mf;
-  mf.initialize(mf_data);
+  mf.initialize(std_cxx11::make_shared<MatrixFree<dim,number> >(mf_data));
```

Comment:
No problem. I used `make_shared` because there was less code to change in the tests but you are right. We don't want someone who looks at the tests to get a bad habit.